app/src/main/res/values/strings.xml

```diff
@@ -14,4 +14,14 @@
         venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla
         mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,</string>
     <string name="collpasing_inc_header_title">HEADER</string>
+    <string name="view_pager_title">Header Title</string>
+    <string name="filtro">Filter</string>
```

Comment:
could you name the resource `filter` instead of `filtro`? so everything is in English ;-)